js-miniapp-bridge/src/types/token-data.ts

```diff
@@ -0,0 +1,5 @@
+/** Token data type. */
+export interface TokenData {
+  token: string;
+  validUntil: number;
```

Comment:
Maybe we can return this as a `Date` object? It would be more convenient for the users and would ensure that the bridge is receiving the same date format on both platforms. What do you think?

Reply:
I think the `long` type is more generic for cross-platforms. @ts-leojoseph-r  What do you think?

Reply:
Sorry, I think I commented this in the wrong place - I agree it's best to return a `long` over the JS bridge. But I think we could convert it to a `Date` before returning the object to the user. But I guess that might require two different types to be defined - one for the object received over the bridge and one for the object returned to the user.

However, if we return the date as a `number`, then we need to add information in the docs about the format of the date, i.e. number of milliseconds since the Unix Epoch.

Reply:
I guess `long` type is okay but at the same time it is good to convert the same to `Date`